src/Control/Functor/Linear/Internal/Class.hs

```diff
@@ -123,6 +143,28 @@ class (Data.Applicative f, Functor f) => Applicative f where
 dataPureDefault :: Applicative f => a -> f a
 dataPureDefault x = pure x
 
+instance Monoid a => Applicative ((,) a) where
```

Comment:
Why can't we derive it through `Generically1` here as you do in `Data.Functor.Linear.Internal.Applicative`?

Reply:
I'm sure we can.

Reply:
Oh, no we can't. These are *control* applicatives. I don't think there's a useful generic way to define them.